### PR TITLE
Rb 4.1.x

### PR DESCRIPTION
**Pull request by ian57**
Please make sure your PR is ready to be merged !
- [ ] You added the changes in CHANGELOG.md
- [ ] You choose the right repository branch to make the PR
- [ ] You described the PR as below

Changes :
- Changing dosbox version -> r3970
## \- Changing SVN site 

Related to (put here the others PR in other repositories)

### CODE REVIEW

**Comment by digitalLumberjack** [comment]
o ok :)

**Comment by digitalLumberjack** [comment]
Hi @ian57 could you change the CHANGELOG.md describing your changes please :)

**Comment by ian57** [comment]
Ok, but where should I write this... in my changelog.md here https://github.com/ian57/recalbox-buildroot/blob/rb-4.1.X/CHANGELOG.md or somewhere else?

**Comment by digitalLumberjack** [comment]
Yes in changelog here, but this changelog must be merged with 4.0.X before :) I'll make the merge

**Comment by ian57** [comment]
Argggllll sorry!!! too much to learn before doing things in the right way.

**Comment by digitalLumberjack** [comment]
I merged the 4.0 in 4.1, you can now your changes under Unreleased section :)

File: configs/recalbox-rpi2_defconfig [hunk 1]
```diff
@@ -159,6 +159,8 @@ BR2_PACKAGE_LIBRETRO_PROSYSTEM=y
 BR2_PACKAGE_LIBRETRO_TGBDUAL=y
 BR2_PACKAGE_LIBRETRO_MGBA=y
 BR2_PACKAGE_LIBRETRO_MAME2003=y
+BR2_PACKAGE_LIBRETRO_REICAST=y
```
Comment (digitalLumberjack) [inline configs/recalbox-rpi2_defconfig:167]: seems something is wrong with this :)

**Comment by substring** [comment]
@ian57 @digitalLumberjack maybe we should close this PR ? it looks like it's a bit messy and not really appropriate ;)

**Pull request closed without merging**